openreview/venue/process/invitation_edit_process.py

```diff
@@ -95,6 +95,8 @@ def update_note_readers(submission, paper_invitation):
 
         def updated_content_readers(note, paper_inv):
             updated_content = {}
+            if 'content' not in paper_inv.edit['note']:
```

Comment:
I am checking if the invitation has a content here, if not then the date process does not try to update the note.

Reply:
looks good